contracts/nexus-gateway/src/contract/migrations/v1_0_0.rs

```diff
@@ -0,0 +1,127 @@
+#![allow(deprecated)]
+
+use axelar_wasm_std::address;
+use axelar_wasm_std::error::ContractError;
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{Addr, DepsMut};
+use cw_storage_plus::Item;
+
+use crate::contract::{CONTRACT_NAME, CONTRACT_VERSION};
+use crate::msg::MigrateMsg;
+use crate::state;
+
+const BASE_VERSION: &str = "1.0.0";
+const CONFIG: Item<Config> = Item::new("config");
+
+pub fn migrate(deps: &mut DepsMut, msg: MigrateMsg) -> Result<(), ContractError> {
```

Comment:
why use a &mut instead of the owned type?

Reply:
oh cuz I initially was setting the contract version in the contract, but decided later to move it inside for the sake of easier testing. Will make it take over the ownership.